packages/headless/src/controllers/core/context/headless-context-reserved-keys.ts

```diff
@@ -0,0 +1,32 @@
+//const ReservedContextKeys = [] as const;
```

Comment:
Is this normal
```suggestion
```

Reply:
Yep: it's what to use after the example is the example is no longer needed